### PR TITLE
OP-401 increase coverage in supplier, format Manager file

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Coverage before:
image

Coverage after:
image

### CODE REVIEW

File: src/test/java/org/isf/supplier/test/Tests.java [hunk 1]
```diff
@@ -77,28 +81,100 @@ public void testSupplierSaveOrUpdate() throws Exception {
...
+	@Test
+	public void testMgrGetHashMap() throws Exception {
+		int code = _setupTestSupplier(false);
+		Supplier foundSupplier = supplierBrowserManager.getByID(code);
+		// get all (including deleted)
+		HashMap<Integer, String> allSuppliers = supplierBrowserManager.getHashMap(true);
+		// get all (not including deleted)
+		HashMap<Integer, String> suppliers = supplierBrowserManager.getHashMap(true);
+		assertThat(allSuppliers).isEqualTo(suppliers);
+	}
```
Comment (emecas) [inline src/test/java/org/isf/supplier/test/Tests.java:145]: I am not sure what are you attempting here,  the HashMaps asserted as equal `allSuppliers ` and `suppliers`  are both coming from a call to same method, with same `true` parameter provided, and no logic between calls.

Also `foundSupplier`  is not being used.

**Review by emecas — APPROVED** [review]
LGTM, but I did not get your idea for `testMgrGetHashMap`

Thanks @dbmalkovsky 

```
Results :

Tests run: 759, Failures: 0, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:19 min
[INFO] Finished at: 2021-01-14T19:13:22-05:00
[INFO] ------------------------------------------------------------------------
```

File: src/test/java/org/isf/supplier/test/Tests.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/test/java/org/isf/supplier/test/Tests.java]: @emecas Good catch.  I've updated the test after correcting the *getHashMap()* method in the manager class 😄 (see this [JIRA issue](https://openhospital.atlassian.net/browse/OP-407) for more information).

File: src/main/java/org/isf/supplier/manager/SupplierBrowserManager.java [hunk 2]
```diff
@@ -27,43 +27,46 @@
...
 	 */
 	public HashMap<Integer, String> getHashMap(boolean all) throws OHServiceException {
-		List<Supplier> supList = getAll();
-		HashMap<Integer, String> supMap = new HashMap<Integer, String>();
+		List<Supplier> supList;
+		if (all) {
+			supList = ioOperations.getAll();
+		} else {
+			supList = ioOperations.getList();
+		}
```
Comment (emecas) [inline src/main/java/org/isf/supplier/manager/SupplierBrowserManager.java:68]: Makes sense now, good job

File: src/test/java/org/isf/supplier/test/Tests.java [hunk 3]
```diff
@@ -77,28 +81,106 @@ public void testSupplierSaveOrUpdate() throws Exception {
...
+		// get all (not including deleted)
+		HashMap<Integer, String> suppliers = supplierBrowserManager.getHashMap(false);
+		assertThat(allSuppliers).isEqualTo(suppliers);
+		// "delete" a supplier
+		foundSupplier.setSupDeleted('Y');
+		supplierBrowserManager.saveOrUpdate(foundSupplier);
+		// get all (not including delete)
+		suppliers = supplierBrowserManager.getHashMap(false);
+		assertThat(allSuppliers).isNotEqualTo(suppliers);
+	}
```
Comment (emecas) [inline src/test/java/org/isf/supplier/test/Tests.java:151]: 😎

**Review by emecas — APPROVED** [review]

**Review by mwithi — APPROVED** [review]
Thank you a lot guys!
```
Results :

Tests run: 939, Failures: 0, Errors: 0, Skipped: 1
```

**Pull request merged**